packages/gguf/src/gguf.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { describe, expect, it } from "vitest";
+import { GGMLQuantizationType, gguf } from "./gguf";
+
+const URL_LLAMA = "https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q2_K.gguf";
+const URL_MISTRAL_7B =
+	"https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.2-GGUF/resolve/main/mistral-7b-instruct-v0.2.Q5_K_M.gguf";
+const URL_GEMMA_2B = "https://huggingface.co/lmstudio-ai/gemma-2b-it-GGUF/resolve/main/gemma-2b-it-q4_k_m.gguf";
```

Comment:
nit: in subseq PR, we can probably move them under https://huggingface.co/huggingfacejs org so that the tests still work/pass if oweners of those repos make any change to those files or filenames

Reply:
or just replace main with the current revision?

Reply:
There's also https://huggingface.co/datasets/huggingface/moon-tests (which we could move to a resource group :) )

Reply:
getting the current versions of models https://github.com/huggingface/huggingface.js/pull/540/commits/007c451bb94021d6a181b0523988cc9ac1b9b6fa